tests/InstrEngineTests/InstrEngineTests/EmbeddedResources/DynamicCodeTests.cs

```diff
@@ -45,22 +49,57 @@ static void Main(string[] args)
 
         private static void CallCompiledAssembly()
         {
-            string[] assemblies = { "System.dll" };
-            CompilerParameters cp = new CompilerParameters(assemblies);
-            cp.OutputAssembly = "DynamicCodeAssembly.dll";
-            cp.GenerateInMemory = true;
-            CodeDomProvider provider = CodeDomProvider.CreateProvider("CSharp");
-            CompilerResults results = provider.CompileAssemblyFromSource(cp, CallForwarderSource);
-
-            Type t = results.CompiledAssembly.GetType("CompiledCode.CallForwarder");
+            SourceText sourceText = SourceText.From(CallForwarderSource);
```

Comment:
Rather than having this debuggee invoke Roslyn to compile the assembly dynamically, I think I'll have it precompiled by the test. My understanding is that the CallCompiledAssembly test is focused on making such that assemblies that are loaded after the initial managed load can be instrumented. This type of change will simplify this test such that (1) Roslyn is not needed by the test anymore, (2) additional compiled-time references do not need to be calculated, and (3) the static config can be removed.